unified_planning/grpc/proto_reader.py

```diff
@@ -0,0 +1,544 @@
+# Copyright 2021 AIPlan4EU project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import fractions
+from typing import OrderedDict
+from unified_planning.exceptions import UPException
+
+import unified_planning.grpc.generated.unified_planning_pb2 as unified_planning_pb2
+import unified_planning.model
+import unified_planning.plan
+from unified_planning.grpc.converter import Converter, handles
+from unified_planning.model import (
+    DurativeAction,
+    Effect,
+    InstantaneousAction,
+    Parameter,
+    Problem,
+    Variable,
+)
+from unified_planning.model.effect import EffectKind
+from unified_planning.model.operators import OperatorKind
+from unified_planning.shortcuts import BoolType, IntType, RealType, UserType
```

Comment:
The `shortcuts` should be used only by an end user but not inside the `unified-planning`